api/src/main/java/org/apache/gravitino/rel/expressions/distributions/Strategy.java

```diff
@@ -50,7 +50,10 @@ public enum Strategy {
   RANGE,
 
   /** Distributes data evenly across partitions. */
-  EVEN;
+  EVEN,
+
+  /** Distributes data randomly across partitions or table. */
+  RANDOM;
```

Comment:
What are the differences between `EVEN` and `RANDOM`?

AFAIK, `RANDOM` is a kind of implementation of `EVEN`

Reply:
Both aim to balance the distribution of data to optimize performance, "Random" emphasizes more on the randomness of the data, while "Even" focuses on maintaining the uniformity of the distribution.

They are slightly different.

Reply:
@FANNG1 do you have any comments on this issue.

Reply:
IMO,  `HASH`, `RANGE` `RANDOM` are the implemetation how we do the distribution,  `even` is the something like distribution result,  both `HASH` and `RANDOM` are the implementation of  `EVEN`

Reply:
So it's recommend to remove `EVEN`? But I remember that @yuqi1129 has done research and there is a certain kind of table that uses `EVEN` as a distribution name

Reply:
I don't think HASH and RANDOM are the implementation of EVEN. In most cases, the Hash strategy will not even distribute the data, for example, if the hash function is the last digit of the key and the last digit of the key is not uniformly distributed. In this case, the data will not be evenly distributed.

 EVEN distributions, in my opinion, round-robin strategy is the best one that fits the EVEN strategy.

Reply:
@yuqi1129 , do you remember which kind of table use even distribution, could it be replaced by round-robin or random?

Reply:
Please refer to 
https://docs.aws.amazon.com/redshift/latest/dg/r_CREATE_TABLE_NEW.html#r_CREATE_TABLE_usage-examples
https://learn.microsoft.com/en-us/sql/t-sql/statements/create-table-azure-sql-data-warehouse?view=aps-pdw-2016-au7

Reply:
@jerryshao any thoughts on this point? #4991 depends on this one.

Reply:
We could not reach an agreement till now, so I postponed it to 0.7.0 as it's not a bugfix.